WooCommerce/Classes/ViewRelated/Products/Categories/ProductCategoryListViewController.swift

```diff
@@ -6,10 +6,12 @@ import WordPressUI
 /// Please note that this class cannot be used as is, since there is not configuration for navigation.
 /// Instead, it shall be embedded through view controller containment by adding it as a child to other view controllers.
 ///
-final class ProductCategoryListViewController: UIViewController {
+final class ProductCategoryListViewController: UIViewController, GhostableViewController {
 
     @IBOutlet private var tableView: UITableView!
-    private let ghostTableView = UITableView()
+
+    lazy var ghostTableViewController = GhostTableViewController(options: GhostTableViewOptions(displaysSectionHeader: false,
```

Comment:
Nit: should we also set the default value for `displaysSectionHeader` to be `false`? I see that's the most common case too.

Reply:
Yeah, it is the most common. I considered that but in this case, I left it because in the underlying `GhostOptions` from `WordPressUI` they have true as the default value for `displaysSectionHeader`, so I wanted them to be consistent as much as possible 😄